src/v/raft/heartbeat_manager.cc

```diff
@@ -74,12 +74,8 @@ heartbeat_manager::heartbeat_requests heartbeat_manager::requests_for_range() {
         }
 
         for (auto& [id, follower_metadata] : r->_fstats) {
-            if (follower_metadata.has_inflight_appends()) {
```

Comment:
Hmm will this result in full heartbeat spam if lw heartbeats are disabled and there are appends in flight?